commands/apps/index.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+let co  = require('co');
+let cli = require('heroku-cli-util');
+let _   = require('lodash');
+
+function* run (context, heroku) {
+  let org = (!context.flags.personal && !context.flags.all && context.org) ? context.org : null;
+  let space = context.flags.space;
+
+  function regionizeAppName (app) {
+    if (app.region && app.region.name !== 'us') {
+      return `${app.name} (${cli.color.green(app.region.name)})`;
+    } else {
+      return app.name;
+    }
+  }
+
+  let isOrgApp = app => !app.owner.email.endsWith('@herokumanager.com');
```

Comment:
@dickeyxxx could you change the name of this, as it returns false if the app is an org app?